src/components/UI/Header.js

```diff
@@ -40,4 +41,17 @@ Header.propTypes = {
   authentication: PropTypes.object.isRequired,
 };
 
-export default Header;
+const mapStateToProps = state => {
+  return {
+    authentication: state.authentication,
+  };
+};
+
+const mapDispatchToProps = dispatch => {
+  return {};
+};
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
```

Comment:
if you don't have any actioins to dispatch you can leave this argument as `null`